pylops/avo/prestack.py

```diff
@@ -234,11 +236,13 @@ def PrestackWaveletModelling(
     wavc : :obj:`int`, optional
         Index of the center of the wavelet
     vsvp : :obj:`np.ndarray` or :obj:`float`, optional
-        VS/VP ratio
+        :math:`V_S/V_P` ratio
     linearization : :obj:`str`, optional
-        choice of linearization, ``akirich``: Aki-Richards,
-        ``fatti``: Fatti, ``ps``: PS, or any function on the form of
-        ``pylops.avo.avo.akirichards``
+        * "akirich": Aki-Richards
+
+        * "fatti": Fatti
+
+        * "PS": PS, or any function on the form of ``pylops.avo.avo.akirichards``
```

Comment:
I see that the original description may have been misleading. It should be

* "akirich": Aki-Richards

* "fatti": Fatti

* "PS": PS, 

* any function on the form of ``pylops.avo.avo.akirichards``

Reply:
Oh, I see! Changed in the newest commit.